tests/requests/test_favorites_requests.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from IPython import embed
+from sqlalchemy import create_engine
+from app.database import Base
+from app.main import app
+import pytest
+from fastapi.encoders import jsonable_encoder
+from app.database import SessionLocal
+from app import crud
+from app.schemas import MarketCreate, UserCreate, FavoriteCreate
+from os import getenv
+
+engine = create_engine(getenv("DATABASE_URL"))
+Base.metadata.drop_all(bind=engine)
+Base.metadata.create_all(bind=engine)
+
+@pytest.fixture
```

Comment:
We could also pull this out into a conftest.py in later refactoring pass throughs